# ==============================================================================
# FIXING COMMIT DIFF
# ==============================================================================
# File: docshell/base/WindowContext.cpp
# Commit: 13a9d145a550
# Full Hash: 13a9d145a55053585dd7477bdc8b7842606784a6
# Author: Toshihito Kikuchi <tkikuchi@mozilla.com>
# Date: 2021-12-17 03:51:53
# Description:
#   Bug 1743663 - Remove windowContext.hasActivePeerConnections.  r=peterv
#   
#   TabUnloader.jsm can directly call windowGlobal.hasActivePeerConnections(),
#   so windowContext.hasActivePeerConnections is not needed.
#   
# ==============================================================================

diff -r 630013ad7f37 -r 13a9d145a550 docshell/base/WindowContext.cpp
--- a/docshell/base/WindowContext.cpp	Thu Dec 16 15:44:15 2021 +0000
+++ b/docshell/base/WindowContext.cpp	Thu Dec 16 16:14:13 2021 +0000
@@ -68,12 +68,6 @@
   return mBrowsingContext->mCurrentWindowContext == this;
 }
 
-bool WindowContext::HasActivePeerConnections() {
-  MOZ_ASSERT(TopWindowContext() == this,
-             "HasActivePeerConnections is set only in the top window context");
-  return Canonical()->HasActivePeerConnections();
-}
-
 bool WindowContext::IsInBFCache() {
   if (mozilla::SessionHistoryInParent()) {
     return mBrowsingContext->IsInBFCache();